x.toml

```diff
@@ -156,3 +169,38 @@ members = [
     "secure/key-manager",
     "storage/backup/backup-cli",
 ]
+
+# ---
+# Determinator rules
+# ---
+
+# CI-related files. TODO: maybe have separate rules for local and CI?
+[[determinator.path-rule]]
+globs = [".github/**/*", ".circleci/**/*", "codecov.yml"]
+mark-changed = "all"
+
+# Core devtools files.
+[[determinator.path-rule]]
+globs = ["cargo-toolchain", "cargo-flags", "scripts/**/*", "x.toml"]
```

Comment:
not sure if I agree with scripts/ being in the all rule.   I would think scripts should be orthogonal except for dev_setup.sh in CI.   Yeah just checked them all think only dev_setup.sh in CI should spark a full rebuild.

Reply:
Ah cool, I can do that then.

Reply:
Done.